src/Aquifer.Data/Schemas/BibleLicenseInfoSchema.cs

```diff
@@ -1,6 +1,6 @@
-﻿namespace Aquifer.API.Endpoints.Bibles;
+namespace Aquifer.Data.Schemas;
 
-public sealed class BibleLicenseInfo
+public sealed class BibleLicenseInfoSchema
```

Comment:
Thoughts on dropping the `Schema` suffix?  These types will be the same name in the auto-generated client.